docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md

```diff
@@ -93,7 +93,7 @@ The `DockerFile` is commonly placed in the root folder of your app or service an
 
 In Figure 4-24, you can see how a docker-compose file is added by using the Docker Extension for VS Code.
 
-![Console view of Docker extension for VS Code.](./media/image24.png)
+![Console view of Docker extension for VS Code.](./media/docker-apps-inner-loop-workflow/add-docker-files-to-workspace-command.png)
```

Comment:
Weird, this seems like the image doesn't match the description. I filed an issue for the author.

Reply:
Hm … I see that now. I was working with the assumption that, if AltText existedfor a file, then I could proceed. Would you like me to verify that the ALtText is applicable as part of this effort, or is this more of a one aff anomaly?

Reply:
I think this is just an anomaly